spec/playlist_spec.rb

```diff
@@ -194,4 +194,20 @@
     expect(playlist.master?).to be true
     expect(playlist.items.size).to eq 6
   end
+
+  it 'should return the total duration of a playlist' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.